docs/reference/accessibility/orca-preferences/orca-preferences-pronunciaton.md

```diff
@@ -0,0 +1,24 @@
+
+
+# Pronunciation preferences
```

Comment:
Isn't this actually a how-to guide? Something like "Override Orca's pronunciation of words"?